webv2/web.go

```diff
@@ -1739,6 +1747,8 @@ func createStreamingCfgFile(sessionState *session.SessionState, targetDetails ta
 		DataflowCfg: DataflowCfg{
 			JobName:  "",
 			Location: sessionState.Region,
+			Network: dataflowConfig.Network,
+			Subnetwork: dataflowConfig.Subnetwork,
```

Comment:
nit: is subnetwork a term often used on GCP ? If not, subnet seems like the more commonly used name for this.

Reply:
Yes, subnetwork is often used in GCP. 

Ref - https://cloud.google.com/dataflow/docs/guides/specifying-networks